command/ssh/login.go

```diff
@@ -24,9 +24,9 @@ func loginCommand() cli.Command {
 		Name:   "login",
 		Action: command.ActionFunc(loginAction),
 		Usage:  "adds a SSH certificate into the authentication agent",
-		UsageText: `**step ssh login** <identity>
+		UsageText: `**step ssh login** [<identity>]
 [**--token**=<token>] [**--provisioner**=<name>] [**--provisioner-password-file**=<file>]
```

Comment:
Can we add an identity flag to the usage text

Reply:
The identity flag is only there so it can be defined in the defaults.json, you're not supposed to use it.